core/embed/rust/src/ui/model_tr/layout.rs

```diff
@@ -256,7 +256,11 @@ fn content_in_button_page<T: Component + Paginate + MaybeTrace + 'static>(
     // Left button - icon, text or nothing.
     let cancel_btn = if let Some(verb_cancel) = verb_cancel {
         if !verb_cancel.is_empty() {
-            Some(ButtonDetails::text(verb_cancel))
+            if verb_cancel.as_ref() == "left_arrow_icon" {
```

Comment:
Nit: change to `"<"` for consistency with model T.

Reply:
Changed in [9432da1](https://github.com/trezor/trezor-firmware/pull/3188/commits/9432da1bdb84e6739f1b011164d8aebdc955838a)